turbinia/state_manager.py

```diff
@@ -33,6 +34,12 @@
 from turbinia import TurbiniaException
 
 config.LoadConfig()
+if 'unittest' in sys.modules.keys():
```

Comment:
Aren't we by default going to be testing either with a state_manager of datastore or redis and can just import missing libraries there? Seems like this would cause a double import, but guess Python has its own way of handling that

Reply:
Yes and no. When we run ```run_tests.py``` this will run all tests, which executes both datastore and redis tests. Not simultaneously but the tests would need to import the right gcp/redis modules.

Those import statements below line 37 would only be executed if the unittest module is loaded, which only happens when running the unittests either via run_tests.py or manually using python -m unittest. From my testing, double import doesn't happen and the unit tests fail unless you import gcp and redis, possibly due to the way unittest executes the imports. There might be a better way of doing this but i couldn't find one.

Reply:
Ack. Per our chat this makes much more sense to leave as is given depending on which Turbinia config gets pulled, the state_manager can be different in tests and may pass or fail depending on what config file is locally available hence having to import all under unittests.